tests/modules/maxquant/lfq/main.nf

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { MAXQUANT_LFQ } from '../../../../modules/maxquant/lfq/main.nf' addParams( options: [:] )
+
+workflow test_maxquant_lfq {
+        
+    input = [ [ id:'test' ], // meta map
+              file(params.fasta, checkIfExists: true), file(params.paramfile, checkIfExists: true) 
+            ]
+
+
+    rawfiles = [file('https://github.com/wombat-p/MaxQuant-Workflow/raw/dev/Nextflow/data_test/OVEMB150205_12.raw') , file('https://github.com/wombat-p/MaxQuant-Workflow/raw/dev/Nextflow/data_test/OVEMB150205_14.raw') ]
```

Comment:
could you please add the test data to the modules branch of `nf-core/testdata` repository? 
Then, please reference the test data from https://github.com/nf-core/modules/blob/master/tests/config/test_data.config and use the entries from the config file for the tests.

Reply:
Did that, and waiting to get the data into the `testdata` repo.